docs/java/features/odata/overview.md

```diff
@@ -31,4 +38,51 @@ The Cloud SDK supports consumption of OData services through the following featu
 - A generator (available as command line interface and Maven plugin) that is capable of generating typed OData clients from service definitions (OData metadata documents)
 - Java Virtual Data Model (VDM): pre-generated typed OData client for [all OData services of SAP S/4HANA Cloud](https://api.sap.com/package/SAPS4HANACloud?section=Artifacts)
 
-The SDK supports both OData v2 and OData v4 protocols. However, parts of the v4 implementation are still in beta and not all features are supported yet.
+The SDK supports both OData v2 and OData v4 protocols.
+
+## OData features supported by SAP Cloud SDK for Java
```

Comment:
In general I like the idea of a feature matrix. I would structure it a bit more to make it easier to find stuff. 

I could imagine for instance:

- Operations (CRUDQ Count Functions Actions Batch)
- Query Parameters (select, filter, top, skip etc.)
- Nested Queries
- Filter Functions
- Deep Operations (targeting Navigational Properties directly)

This also gives some context to the terms which are easily mistaken or confused as we know from our own expeirience.

Reply:
A valuable addition might also be to add links for these sections to the documentation